core/templates/domain/collection/collection-node.model.ts

```diff
@@ -59,7 +59,7 @@ export class CollectionNode {
   // Returns the title of the exploration represented by this collection node.
   // This property is immutable. The value returned by this function is
   // null if doesExplorationExist() returns false.
-  getExplorationTitle(): string {
+  getExplorationTitle(): string | null {
```

Comment:
Do we actually need for this method to return `null`? Check the places where this method is used and if we somehow utilize null there.

Reply:
In the method definition, it returns a string only if the function gives true else it directly returns null. Therefore the return type of method needs to be both string and null.
![image](https://user-images.githubusercontent.com/68547101/96961502-7327ed80-1522-11eb-8b06-20de837b1331.png)

Reply:
I know that but in the places where the method is used do we actually utilize this feature (that it sometimes returns `null`)?

Reply:
I have removed the if else statement block and have returned this.explorationSummaryObject. I don't think it is breaking the code away. Can  you please see it?

Reply:
Yep, looks fine.